main/modes/mode_diceroller.c

```diff
@@ -624,10 +652,249 @@ void drawRegularPolygon(int xCenter, int yCenter, int8_t sides, float rotDeg, in
     free(vertices);
 }
 
+void oddEvenFillFix(display_t* disp, int x0, int y0, int x1, int y1,

```

Comment:
This should probably just replace `oddEvenFill()` rather than being a separate function.

Reply:
I've cleaned the function up a bit and replaced the original in bresenham.c